Makefile.toml

```diff
@@ -49,6 +49,7 @@ dependencies = [
     "test-docs-default",
     "test-docs",
     "bincode-gen-testdata",
+    "testdata-check",
```

Comment:
@Manishearth I was going to put this in the "tidy" job but I put it in the "test" job instead since it requires building most of ICU4X.  Thoughts?